yarn.lock

```diff
@@ -6588,7 +6588,7 @@ nice-try@^1.0.4:
   resolved "https://registry.yarnpkg.com/nice-try/-/nice-try-1.0.5.tgz#a3378a7696ce7d223e88fc9b764bd7ef1089e366"
   integrity sha512-1nh45deeb5olNY7eX82BkPO7SSxR5SSYJiPTrTdFUVYwAl8CKMA5N9PjTYkHiRjisVcxcQ1HXdLhx2qxxJzLNQ==
 
-node-abi@^2.7.0, node-abi@^3.8.0:
+node-abi@^2.7.0, node-abi@^3.3.0, node-abi@^3.8.0:
```

Comment:
This was as a result of running `yarn` in root directory; seems like we need to update some sub-dependencies.